dot/network/host_test.go

```diff
@@ -54,7 +56,7 @@ func TestConnect(t *testing.T) {
 		NoMdns:      true,
 	}
 
-	nodeB, _, _ := createTestService(t, configB)
+	nodeB, _, _ := createTestService(t, configB, blockState)
```

Comment:
two nodes should not be sharing the same blockState

Reply:
sure